libpkg/pkg_event.c

```diff
@@ -1025,8 +1025,7 @@ pkg_emit_progress_tick(int64_t current, int64_t total)
 	ev.e_progress_tick.current = current;
 	ev.e_progress_tick.total = total;
 
-	pkg_emit_event(&ev);
-
+	return !!pkg_emit_event(&ev);
```

Comment:
this looks like a bool, but our return value is `int`

Reply:
User's event handler may return whatever it wants, but for progress_tick we only care about it being non-zero. So I normalize it to either `0` or `1`.